recipe/build.sh

```diff
@@ -2,8 +2,12 @@ export PYTHONDONTWRITEBYTECODE=1
 pip install --no-deps --no-build-isolation .
 # Conflicts with jupyter_core which is a transitive dep.
 cd $SP_DIR
-rm jupyter.py
-rm -f jupyter.pyc
+if [[ -f jupyter.py ]]; then
+   rm jupyter.py
+fi
+if [[ -f jupyter.pyc ]]; then
+   rm -f jupyter.pyc
+fi
 if [[ -d __pycache__ ]]; then
    rm -rf __pycache__
 fi
```

Comment:
jupyter 1.0.0 used to ship a python file jupyter.py. 
This is not the case with this version. This code can be removed.